src/OmniSharp.Abstractions/Models/v2/SemanticHighlight/SemanticHighlightClassification.cs

```diff
@@ -20,43 +20,42 @@ public enum SemanticHighlightClassification
         Punctuation,
         VerbatimStringLiteral,
         StringEscapeCharacter,
-        ClassName,
-        DelegateName,
-        EnumName,
-        InterfaceName,
-        ModuleName,
-        StructName,
-        TypeParameterName,
-        FieldName,
-        EnumMemberName,
-        ConstantName,
-        LocalName,
-        ParameterName,
-        MethodName,
-        ExtensionMethodName,
-        PropertyName,
-        EventName,
-        NamespaceName,
-        LabelName,
-        XmlDocCommentAttributeName,
+        Class,
+        Delegate,
+        Enum,
+        Interface,
+        Struct,
+        TypeParameter,
+        Field,
+        EnumMember,
+        Constant,
+        Variable,
+        // TODO: Figure out how to get Variable.Readonly to work
+        Parameter,
+        Method,
+        Property,
+        Event,
+        Namespace,
+        Label,
+        XmlDocCommentAttribute,
         XmlDocCommentAttributeQuotes,
         XmlDocCommentAttributeValue,
         XmlDocCommentCDataSection,
         XmlDocCommentComment,
         XmlDocCommentDelimiter,
         XmlDocCommentEntityReference,
-        XmlDocCommentName,
+        XmlDocComment,
         XmlDocCommentProcessingInstruction,
         XmlDocCommentText,
-        XmlLiteralAttributeName,
+        XmlLiteralAttribute,
         XmlLiteralAttributeQuotes,
         XmlLiteralAttributeValue,
         XmlLiteralCDataSection,
         XmlLiteralComment,
         XmlLiteralDelimiter,
         XmlLiteralEmbeddedExpression,
         XmlLiteralEntityReference,
-        XmlLiteralName,
+        XmlLiteral,
```

Comment:
Please revert the changes to the Xml DocComment and Literal classification names. They are taken straight from Roslyn. 
https://github.com/dotnet/roslyn/blob/59727ee4103c75edb799bb46e081427a2e1940ef/src/Workspaces/Core/Portable/Classification/ClassificationTypeNames.cs#L55-L73